NICE.Identity.Management/ClientApp/src/views/Organisation/Organisation.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from "react";
+
+export class Organisation extends Component {
+	render(): JSX.Element {
+		return (
+			<div>
+				<p>Someting else here</p>
```

Comment:
What's this all about?

Reply:
I'd started building other pages as I thought they were included in this card, but they had been extracted out. I left this code in, so that the other developers could build this page and branch off it. Completely not ideal, but it's a long story. Should I just delete this and then sort the conflict when the other branch gets merged in?

Reply:
Not sure I understand entirely...but up to you what you do with it!